apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+#include "CANMsg.h"
+#include <cstdlib>
+
+
+TutorialServo tutorialServo(PA_1);
+CANBus can1(CAN1_RX, CAN1_TX);
+std::Thread rxCANProcessorThread(osPriorityAboveNormal);
+
+// Since the arm app used threads I attempted to use it too,
+// but for this application probably would just put it in the main loop
+void updateServoPosition() {
```

Comment:
u dont have to do this but its usually nice to seperate function declaration and definition. our repo doesnt do this but this is a good convention

Reply:
Ok sure sounds good, I'll separate it from now on